resources/kiali/templates/deployment.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "kiali.name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "kiali.name" . }}
+    chart: {{ template "kiali.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.deployment.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "kiali.name" . }}
+  template:
+    metadata:
+      name: {{ template "kiali.name" . }}
+      labels:
+        app: {{ template "kiali.name" . }}
+        chart: {{ template "kiali.chart" . }}
+        heritage: {{ .Release.Service }}
+        release: {{ .Release.Name }}
+      annotations:
+        sidecar.istio.io/inject: "false"
```

Comment:
Any reason we need to disable istio sidecar?

Reply:
I think all that observability tools should not be inside the service-mesh, no application should have a need to talk to it.
it calls prometheus only which is also outside the mesh. Have no better argument. Do you have an argument why it should be inside?